pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java

```diff
@@ -346,7 +346,7 @@ protected void stopServer() {
     for (BaseServerStarter serverStarter : _serverStarters) {
       serverStarter.stop();
     }
-    FileUtils.deleteQuietly(new File(_baseInstanceDataDir + File.separator + "PinotServer"));
+    FileUtils.deleteQuietly(new File(_baseInstanceDataDir));
```

Comment:
otherwise, we'll see a lot of <tmpDir>/<timestamp> folders left in system temp dir